### PR TITLE
Merge feat warnings into feature branch

### PR DESCRIPTION
**Pull request by ChomickiM**
# Pull Request template

**Link to github issue (if applicable):**

*

**If nothing above, what is your reason for this pull request:**

*

**Changes proposed in this pull request:**

*
*
*

### CODE REVIEW

File: src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java [hunk 1]
```diff
@@ -63,6 +63,9 @@ public void validateConfig(Map<String, ConfigField> descriptorFields, FieldModel
                 if (validationResult.hasErrors()) {
                     fieldErrors.put(key, validationResult.combineErrorMessages());
                 }
+                if (validationResult.hasWarnings()) {
+                    fieldErrors.put(key, validationResult.combineWarningMessages());
```
Comment (jamesrichard91) [inline src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java:67]: Should we be adding warnings to the fieldErrors?

Comment (psantos1113) [reply src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java:67]: This is a work in progress.  We are keeping the same functionality for now and making sure that the code is in place to be able to add warnings even though they are treated as errors. The next branch and refactoring will to convert to the objects that contain information about warnings or errors. Taking a phased approach to this refactor.

Comment (psantos1113) [reply src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java:67]: That's our next step.   We know this code works with the existing code and provides the pieces to add warnings.  Next step is to incorporate all of it together.

Comment (jamesrichard91) [reply src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java:67]: Can we add a TODO in the places that we know need to change?

Comment (psantos1113) [reply src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java:67]: We will add TODOs as we move forward.  We are working on a feature branch so the feature branch will ultimately be reviewed by everyone.

**Pull request merged**